vars/shared-CKANTest.var.yml

```diff
@@ -240,14 +240,6 @@ extensions:
       <<: *CKANExtOIDC
       version: "master"
 
-    CKANExtClamAV:
```

Comment:
so we are dropping this?

Reply:
For now it simply doesn't work and is blocking uploads on the sandbox, which hinders other testing.